grammar.js

```diff
@@ -413,10 +412,10 @@ module.exports = grammar({
       optional(',')
     ),
 
-    expression_list: $ => seq(
+    expression_list: $ => prec.right(seq(
       commaSep1($._expression),
       optional(',')
-    ),
+    )),
```

Comment:
I tried a few things to prevent the `prec.right`, but this `commaSep1($._expression)` has a conflict with the way we model `print_statement` currently. My thinking here is since `print_statement` matches against `expression_list`, the `prec.right` makes `expression_list` slightly more "greedy" and correctly matches against the inner terms of a `print` statement. @maxbrunsfeld if I've gotten the way `prec.right` works turned around in my head, please correct me! :bow: